lib/ansible/plugins/lookup/pmp.py

```diff
@@ -0,0 +1,80 @@
+# (c) 2016, Andrew Zenk <azenk@umn.edu>
+# (c) 2017 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+from ansible.errors import AnsibleError
+from ansible.plugins.lookup import LookupBase
+from ansible.modules.web_infrastructure.pmp import PasswordManagerPro
+
+try:
+    from __main__ import display
+except ImportError:
+    from ansible.utils.display import Display
+
+    display = Display()
+__metaclass__ = type
+
+DOCUMENTATION = """
+    lookup: pmp
+    author:
+      -  Bernat Mut <bernat.mut@aireuropa.com>
+    version_added: "2.5"
+    short_description: Recover password account from Password Manager Pro (PMP)
+    description:
+      - Use this module to  requests passwords from an specific account to PMP
+    options:
```

Comment:
Looks like this was copy-and-pasted from the LastPass lookup plugin, and hasn't yet been updated for Password Manager Pro.